utils/frame/benchmarking-cli/src/bedrock/weights.hbs

```diff
@@ -0,0 +1,58 @@
+//! AUTO GENERATED DO NOT MODIFY
+
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// Storage DB weights for the {{runtime_name}} runtime and {{db_name}}.
+pub mod constants {
+	use frame_support::{parameter_types, weights::{RuntimeDbWeight, constants}};
+
+	parameter_types! {
+		/// Automatically generated weights for {{db_name}}.
+		pub const {{db_name}}Weight: RuntimeDbWeight = RuntimeDbWeight {
+		{{#if read_ns}}
+			/// {{read_ns}} ns
```

Comment:
seems like a bit of a useless comment given the line below :P

Reply:
Updated, see below